### DISCUSSION TITLE
Hope grafana can support Chinese environment

### DISCUSSION THREAD

**Discussion by jinrui-zhu** [grafana/grafana#50571] [General]
As far as I know, more and more Chinese users are using grafana. I believe that the project party can also see the relevant data. It's time to adapt to the Chinese version.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
